src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -80,27 +80,37 @@
     {%  if draft.formaliseRepaymentPlan.option.value === FormaliseRepaymentPlanOption.SIGN_SETTLEMENT_AGREEMENT.value %}
       {{ tableStart('settlementAgreement', 'Settlement agreement') }}
 
-      {% if draft.courtOfferedPaymentIntention.paymentOption === PaymentType.BY_SPECIFIED_DATE.value %}
-        {% set amountToPay = 'the full amount' %}
-        {% if claim.response.responseType == ResponseType.PART_ADMISSION.value %}
-          {% set amountToPay = claim.response.amount | numeral %}
-        {% endif %}
-
-        {{ row(label = 'The offer',
+      {% if draft.courtOfferedPaymentIntention.paymentOption === PaymentType.IMMEDIATELY.value %}
+        {{ row(label = 'The agreement',
+          value = t('{{ defendantFullName }} will pay {{ totalAmount }} immediately.
+        {{ claimantName }} will receive the moment no later than {{ paymentDate }}.
+        Any cheques or transfers will be clear in {{ defendantFullName }} account by this date.',
```

Comment:
Do we need a possessive apostrophe here after the defendant's name?
i.e., instead of "... clear in Mr Foo Bar account ...", say "... clear in Mr Foo Bar's account ..."